cmake/Install.cmake

```diff
@@ -357,6 +357,7 @@ set(RULES_XLS Rules9-5-0-to-9-6-0.md)
 install(FILES "${PROJECT_SOURCE_DIR}/release/Bugreprt.txt" DESTINATION "./")
 install(FILES "${PROJECT_SOURCE_DIR}/release/favicon.png" DESTINATION "./")
 install(FILES "${PROJECT_SOURCE_DIR}/release/readme.html" DESTINATION "./")
+install(FILES "${PROJECT_SOURCE_DIR}/release/Deprecation.html" DESTINATION "./")
```

Comment:
Install command to put the deprecation document in the installer, it's a nice looking document!